media.css

```diff
@@ -0,0 +1,442 @@
+    display: grid;
+    grid-template-columns: auto;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .card1 {
+    display: flex;
+    padding: 24px;
+  }
+
+  .card2 {
+    display: flex;
+    padding: 24px;
+    width: 90%;
+  }
+
+  .project1 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
+    margin: auto;
+    margin-top: 110px;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  .project2 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row-reverse;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
+    margin: auto;
+    margin-top: 110px;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  .project3 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
+    margin: auto;
+    margin-top: 110px;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  .project4 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row-reverse;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
+    margin: auto;
+    margin-top: 110px;
+    margin-bottom: -350px;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    padding: 10px;
+    margin-left: 65px;
+    margin-top: -15px;
+  }
+
+  .para {
+    margin-left: 78px;
+  }
+
+  .btn {
+    margin-left: 72px;
+  }
+
+  .list ul {
+    display: flex;
+    margin-left: 64px;
+    margin-top: -15px;
+    align-items: flex-start;
+  }
+
+  .task {
+    display: flex;
+    flex-direction: column;
+    margin-right: 65px;
+    margin-bottom: 30px;
+  }
+
+  .task ul {
+    display: flex;
+    gap: 12px;
+    border-radius: 8px;
+    margin-right: 50px;
+    color: #05070a;
+    list-style-type: none;
+    text-align: start;
+    font-family: 'Poppins', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+    font-weight: 500;
+    margin-left: 43px;
+  }
+
+  .canopy {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .describe {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 75%;
+    width: 100%;
+
+    /* margin: 30px; */
```

Comment:
I think you should not use code as a comment, delete any extra code which is not required.

Reply:
yes sir !!